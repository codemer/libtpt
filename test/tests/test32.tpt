@foreach x ("a", "b", "c", "d", "e") {
	@<@if (!@compare(${x}, "d"))
		{@last}
	@<@foreach y (1, 2, 3, 4) {
		@<@if (${y} == 2) {@next}
		@<${x} ${y}
	@<}
}
