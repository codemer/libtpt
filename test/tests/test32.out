a 1
a 3
a 4
b 1
b 3
b 4
c 1
c 3
c 4
